nix/haskell.nix

```diff
@@ -0,0 +1,46 @@
+############################################################################
+# Builds Haskell packages with Haskell.nix
```

Comment:
Where does this file come from? Just a template that you use?

Reply:
Yes. This is pretty much the template @iohk-devops have used across repositories.

My ideal setup would be an empty file with a call to `stackProject` or `cabalProejct` (delete all the `.nix` files!!!) but for consistency I've opted to use the same.

Reply:
Okay, but then I want a comment telling me it's a template and where to go to look to check if the template is up to date.

Reply:
So you want a template repository, I guess that would be nice to have...